examples/browser-webrtc/server/.cargo/config.toml

```diff
@@ -0,0 +1,2 @@
+[env]
+RUST_LOG = "debug,libp2p_webrtc,webrtc_sctp=off,webrtc_mdns=off,webrtc_ice=off"
```

Comment:
I think the logger configuration would be better in the code when we initialize the logger.